bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -47,9 +45,12 @@
     private final AgsContextPublisher _agsPub = new AgsContextPublisher();
     private final TelescopeForm _w;
 
-    // Stuff that varies with time
-    private SPTarget        _curPos;
-    private Tuple2<Integer, GuideGroup> _curGroup;
+    // The current selection. Can be None if nothing is selected.
+    private Option<Either<SPTarget, IndexedGuideGroup>> _curSelection = None.instance();
```

Comment:
I wonder if we should default the selection to the base position.  Then there is always something selected.  There's always a base position.  That simplifies some of the logic because you don't have to deal with the `Option` wrapper.

I guess it is likely difficult to make that change though.  Let's consider it for after the merge perhaps.